src/parquet/schema.cc

```diff
@@ -620,6 +620,15 @@ const ColumnDescriptor* SchemaDescriptor::Column(int i) const {
   return &leaves_[i];
 }
 
+int SchemaDescriptor::ColumnIndex(const NodePtr& node) const {
```

Comment:
Sorry, for my low activity on this patch, let me take a closer look tomorrow morning